runelite-client/src/main/java/net/runelite/client/plugins/deathindicator/DeathIndicatorConfig.java

```diff
@@ -0,0 +1,235 @@
+	{
+		return true;
+	}
+
+	@ConfigItem(
+		position = 2,
+		keyName = "deathInfoBox",
+		name = "Death InfoBox",
+		description = "Configures whether or not to show item reclaim timer and death world infobox"
+	)
+	default boolean showDeathInfoBox()
+	{
+		return true;
+	}
+
+	@ConfigItem(
+		position = 3,
+		keyName = "deathOnWorldMap",
+		name = "Mark on World Map",
+		description = "Configures whether or not to show death location on the world map"
+	)
+	default boolean showDeathOnWorldMap()
+	{
+		return true;
+	}
+
+
+
+	// Stored Data
+	@ConfigItem(
+		keyName = "hasDied",
+		name = "",
+		description = "",
+		hidden = true
+	)
+	default boolean hasDied()
+	{
+		return false;
+	}
+
+	@ConfigItem(
+		keyName = "hasDied",
+		name = "",
+		description = ""
+	)
+	void hasDied(boolean hasDied);
+
+	@ConfigItem(
+		keyName = "hasRespawned",
+		name = "",
+		description = "",
+		hidden = true
+	)
+	default boolean hasRespawned()
+	{
+		return true;
+	}
+
+	@ConfigItem(
+		keyName = "hasRespawned",
+		name = "",
+		description = ""
+	)
+	void hasRespawned(boolean hasRespawned);
+
+	@ConfigItem(
+		keyName = "worldMapPointSet",
+		name = "",
+		description = "",
+		hidden = true
+	)
+	default boolean worldMapPointSet()
+	{
+		return false;
+	}
+
+	@ConfigItem(
+		keyName = "worldMapPointSet",
+		name = "",
+		description = ""
+	)
+	void worldMapPointSet(boolean worldMapPointSet);
+
+	@ConfigItem(
+		keyName = "deathMarkerReset",
+		name = "",
+		description = "",
+		hidden = true
+	)
+	default boolean deathMarkerReset()
+	{
+		return false;
+	}
+
+	@ConfigItem(
+		keyName = "deathMarkerReset",
+		name = "",
+		description = ""
+	)
+	void deathMarkerReset(boolean deathMarkerReset);
+
+	@ConfigItem(
+		keyName = "deathWorld",
+		name = "",
+		description = "",
+		hidden = true
+	)
+	default int deathWorld()
+	{
+		return -1;
+	}
+
+	@ConfigItem(
+		keyName = "deathWorld",
+		name = "",
+		description = ""
+	)
+	void deathWorld(int deathWorld);
+
+	@ConfigItem(
+		keyName = "deathLocationX",
+		name = "",
+		description = "",
+		hidden = true
+	)
+	default int deathLocationX()
+	{
+		return -1;
+	}
+
+	@ConfigItem(
+		keyName = "deathLocationX",
+		name = "",
+		description = ""
+	)
+	void deathLocationX(int deathLocationX);
+
+	@ConfigItem(
+		keyName = "deathLocationY",
+		name = "",
+		description = "",
+		hidden = true
+	)
+	default int deathLocationY()
+	{
+		return -1;
+	}
+
+	@ConfigItem(
+		keyName = "deathLocationY",
+		name = "",
+		description = ""
+	)
+	void deathLocationY(int deathLocationY);
+
+	@ConfigItem(
+		keyName = "deathLocationPlane",
+		name = "",
+		description = "",
+		hidden = true
+	)
+	default int deathLocationPlane()
+	{
+		return -1;
+	}
+
+	@ConfigItem(
+		keyName = "deathLocationPlane",
+		name = "",
+		description = ""
+	)
+	void deathLocationPlane(int deathLocationPlane);
+
+	@ConfigItem(
+		keyName = "timeOfDeath",
+		name = "",
+		description = "",
+		hidden = true
+	)
+	default String timeOfDeath()
```

Comment:
Config supports Instant

Reply:
ConfigManager says otherwise

Reply:
Yeah, I'm not sure what Adam meant here either.

Reply:
I believe I've fixed everything else mentioned at this point. I'm going to go ahead and proofread my code again then submit it with storing the death time as a string. If I need to change that later, I'll try to figure it out.

Reply:
This is the last unresolved comment @Adam- . I'm sorry that I didn't clean up the commits to make it look nicer, this is my first PR ever, and I screwed up my branch and can't fix it. I tried for about an hour and fifteen minutes to fix my branch so I could squash some pointless commits but it just wasn't happening. :/ Let me know If I need to do anything else.